### PR TITLE
Release notes for Percona Server 5.7.18-16 with more revision correct…

### PR DESCRIPTION
**Pull request by borys-belinsky**
…ions

### CODE REVIEW

**Review by percona-ysorokin — COMMENTED** [review]
@borys-belinsky For both 5.6 and 5.7, please also add compatibility matrix for lp:1673656
from the commit message
https://github.com/percona/percona-server/commit/39b2941544510f186dfa2f97a42ee7c9a0ae8fd2
In particular, it should be stated clearly, that wildcard certificates are supported on platforms with OpenSSL >= 1.0.2 only.

File: doc/source/release-notes/Percona-Server-5.7.18-16.rst [hunk 1]
```diff
@@ -49,46 +49,49 @@ to error log. Bug fixed :tdbbug:`23`.
...
 
 |Percona FT| did not pass cmake flags on to snappy cmake. Bug fixed
 :tdbbug:`41`.  The progress status for partitioned TokuDB table ALTERs was
 misleading. Bug fixed :tdbbug:`42`.
 
-When connecting to the Aurora cluster end point using SSL, SAN (Subject
-Alternative Name) certificates were ignored.  Bug fixed :bug:`1673656` (upstream
-:mysqlbug:`68052`).
+When a client application connecting to the Aurora cluster end point using SSL
+(``--ssl-verify-server-cert`` option), wildcard and :abbr:`SAN (Subject
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-5.7.18-16.rst]: In 5.7 `--ssl-verify-server-cert ` is considered deprecated. So I would change
`(--ssl-verify-server-cert option)` to `(--ssl-verify-server-cert / --ssl-mode=VERIFY_IDENTITY option)`

**Review by percona-ysorokin — APPROVED** [review]
LGTM

File: doc/source/release-notes/Percona-Server-5.7.18-16.rst [hunk 2]
```diff
@@ -20,16 +20,16 @@ Launchpad <https://launchpad.net/percona-server/+milestone/5.7.18-16>`_
 New Feature
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.18-16.rst:20]: New Features ?

File: doc/source/release-notes/Percona-Server-5.7.18-16.rst [hunk 3]
```diff
@@ -49,46 +49,52 @@ to error log. Bug fixed :tdbbug:`23`.
...
+GLOBAL`` or ``SHOW GLOBAL STATUS`` command. Bug fixed :bug:`1385352`.
 
 When running ``SHOW ENGINE INNODB STATUS``, the ``Buffer pool size, bytes``
 entry contained **0**. BUg fixed :bug:`1586262`.
      
 The synchronization between the LRU manager and page cleaner threads was not
 done at shutdown. Bug fixed :bug:`1689552`.
 
-Spurious ``lock_wait_timeout_thread wakeup`` in ``lock_wait_suspend_thread()``
+Spurious ``lock_wait_timeout_thread`` wakeup in ``lock_wait_suspend_thread()``
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.18-16.rst:90]: Suggestion not implemented

File: doc/source/release-notes/Percona-Server-5.7.18-16.rst [hunk 4]
```diff
@@ -101,3 +107,19 @@ Other bugs fixed:
 :bug:`1684601` (upstream :mysqlbug:`86016`).
 
 
+.. note:: Due to new package dependency,
+   Ubuntu/Debian users should use ``apt-get dist-upgrade``
+   or ``apt-get install percona-server-server-5.7`` to upgrade.
+
+.. _compatibility-matrix:
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.18-16.rst:114]: Like with 5.6, this information must live outside of RNs too

**Pull request merged**